src/settings.py

```diff
@@ -84,6 +84,10 @@ class GlobalSettings:
 
     # option learning parameters
     option_learner = "no_learning"  # "no_learning" or "oracle"
+    mlp_regressor_max_itr = 10000
```

Comment:
let's start a new "section" in settings.py with a comment `# torch model parameters` and move these there, along with the `classifier_*` and `regressor_*` that are currently under `# sampler learning parameters`. Also can you rename some of those to make it clear which ones are using what? Like `learning_rate` can stay as is since everyone uses that, but the `classifier_hid_sizes = [32, 32]` under `# sampler learning parameters` should be renamed to `mlp_classifier_hid_sizes`, similarly with `regressor_hid_sizes` -> `neural_gaus_regressor_hid_sizes`, etc.